### PR TITLE
feature  -load map with trending venues

### PR DESCRIPTION
**Pull request by at-phutran**
- Add Key Google ( clientIdKey, clientSecretKey)
-Parse API Trending Venues
- Creat currentLocation
-Load trending venues in map
- Config and add marker in map

### CODE REVIEW

File: ATFoody/Model/Schema/MyLocationManager.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  Copyright © 2017 Asian Tech. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+
+class MyLocationManager: NSObject {
+    static let shared = MyLocationManager()
+    let locationManager: CLLocationManager = CLLocationManager()
+    var currentLocation = CLLocation(latitude: 37.785834, longitude: -122.406417)
```
Comment (at-daiho) [inline ATFoody/Model/Schema/MyLocationManager.swift]: WARNING - Underscores should be used as thousand separator in large decimal numbers.

Comment (at-daiho) [inline ATFoody/Model/Schema/MyLocationManager.swift]: WARNING - Underscores should be used as thousand separator in large decimal numbers.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

File: ATFoody/Model/LocationManager.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+//  ATFoody
+//
+//  Created by Phu Tran on 11/8/17.
+//  Copyright © 2017 Asian Tech. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+protocol LocationManagerDelegate: class {
```
Comment (at-daiho) [inline ATFoody/Model/LocationManager.swift]: WARNING - Delegate protocols should be class-only so they can be weakly referenced.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 3
Serious: 0
Warnings: 3

File: ATFoody/View/Controller/Home/LocationViewController.swift [hunk 3]
```diff
@@ -7,10 +7,117 @@
...
 import UIKit
+import GoogleMaps
+import RealmS
 
 final class LocationViewController: UIViewController {
+    // MARK: - IBOutlets
+    @IBOutlet weak var myMapView: GMSMapView!
 
+    // MARK: - Properties
+    fileprivate var markers: [GMSMarker] = []
```
Comment (at-quangphu) [inline ATFoody/View/Controller/Home/LocationViewController.swift]: This property should be managed by its view model

File: ATFoody/View/Controller/Home/LocationViewController.swift [hunk 4]
```diff
@@ -7,10 +7,117 @@
...
+            let marker = MarkerMap()
+            marker.position = CLLocationCoordinate2D(latitude: latitude, longitude: longitude)
+            marker.title = trending.name
+            marker.icon = GMSMarker.markerImage(with: .red)
+            marker.map = myMapView
+            markers.append(marker)
+        }
+    }
+
+    class MarkerMap: GMSMarker {
```
Comment (at-quangphu) [inline ATFoody/View/Controller/Home/LocationViewController.swift]: Please, create a new file for define a new class

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

File: ATFoody/View/Controller/Home/LocationViewController.swift [hunk 5]
```diff
@@ -7,10 +7,117 @@
...
+        var zoomLevelMarker: Float = 14
+        var tag: Int = 0
+    }
+}
+
+// MARK: - GMSMapViewDelegate
+extension LocationViewController: GMSMapViewDelegate {
+    func mapView(_ mapView: GMSMapView, didTap marker: GMSMarker) -> Bool {
+        let detailVC = VenueDetailViewController()
+        selectedMarker = marker
```
Comment (at-quangphu) [inline ATFoody/View/Controller/Home/LocationViewController.swift]: Don't init variable like this. 
You can init some properties by its `init()` function instead.
Ex: 
`init(marker: GMSMarker) { }`

**Pull request merged**